ClassLibrary1/Class1.cs

```diff
@@ -24,5 +24,18 @@ public static void ColorBlack()
         {
             Console.BackgroundColor = ConsoleColor.Black;
         }
+        public static void hum2()
```

Comment:
Good example, but hum2 better rename to Hum2 (will be explained in several classes)

Reply:
Its almost done